index.js

```diff
@@ -20,7 +20,70 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    //Iteration 1
+    // Recipe.create(data[0])
+    //   .then((res) => {
+    //     console.log("Data added", res);
+    //   })
+    //   .catch((err) => {
+    //     console.log("Something went wrong", err);
+    //   });
+
+
+    let insertRecipes = Recipe.insertMany(data);
+    // let updateRecipe;
+    // let deleteRecipe;
+
+    insertRecipes
+      .then((res) => {
+        console.log("Data added");
+
+        let updateRecipe = Recipe.findOneAndUpdate({ title: "Rigatoni alla Genovese" }, { $set: { duration: 100 } });
+        let deleteRecipe = Recipe.deleteOne({ title: "Carrot Cake" });
+        let disconnectPromise = Promise.all([updateRecipe, deleteRecipe]);
+
+        updateRecipe
+          .then((res) => {
```

Comment:
the disconnectPromise holds the resolution of both the update and delete promises so running the console.logs separately here instead of inside the .then of disconnectPromise doesn't make much sense. To avoid code redundancy you can just declare and store the promises like you did in lines 41 and 42 and then put the code you want to run inside the .then and .catch of the promise.all